ssr/src/page/post_view/mod.rs

```diff
@@ -121,76 +125,13 @@ pub fn CommonPostViewWithUpdates(
     }
 }
 
-#[component]
-pub fn PostViewWithUpdates(initial_post: Option<PostDetails>) -> impl IntoView {
```

Comment:
I don't want you to remove this right now, we'll still probably use this when local-ssr/local-hydrate features are enabled over the MLFeed until we have testcontainer for off chain, but commenting it out is fine